smsapi/ProxyHTTP.cs

```diff
@@ -70,155 +64,75 @@ public Stream Execute(string uri, NameValueCollection data, RequestMethod method
             Dictionary<string, Stream> files,
             RequestMethod method = RequestMethod.POST)
         {
-            string boundary = "SMSAPI-" + DateTime.Now.ToString("yyyy-MM-dd_HH:mm:ss")
-                + new Random().Next(int.MinValue, int.MaxValue) + "-boundary";
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
 
-            ServicePointManager.SecurityProtocol = _Tls12;
-            var webRequest = WebRequest.Create(baseUrl + uri);
-            webRequest.Method = RequestMethodToString(method);
+            var client = new RestClient(baseUrl + uri);

```

Comment:
Why you've chosen RestSharp? Is HttpClient insufficient? The less dependencies, the easier maintenance in the future.

Reply:
In new version `RestSharp` is wrapping `HttpClient`. By default `HttpClient` is exposing only `async` methods and `RestSharp` allows also to use it in non-async way. It's not an issue to make `HttpClient` non-async but still it's redundant code if we are able to use library that already implements it ;) Also `RestSharp` is really popular library that is really well tested and widely used (from nugets it's 179M downloads at this moment).

Reply:
convinced :)